error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/ExplicitArgumentEnumerationTest.java

```diff
@@ -90,6 +89,23 @@ void identification() {
         .doTest();
   }
 
+  @EnabledForJreRange(min = JRE.JAVA_21)
```

Comment:
Perhaps we want to import this? And directly add it to `StaticImport` list? I think having `JRE` here doesn't add more context. WDYT?